tools/localization/crowdInPostImport.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
@Aerilym do we need this?

Reply:
we do, we want to use this eventually on the CI (once strings are valid on crowdin)

Reply:
I just pushed a change to make that script executable